specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+            "SanDrop",
+            "FraudCleared",
+            "CertificateExpired",
+            "CertificateExpirationWarning",
+            "FraudDocumentationRequired",
+            "Unknown"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CertificateOrderActionType",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "Time at which the certificate action was performed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificateOrderContact": {
+      "type": "object",
+      "properties": {
+        "email": {
+          "type": "string"
+        },
+        "nameFirst": {
+          "type": "string"
+        },
+        "nameLast": {
+          "type": "string"
+        },
+        "phone": {
+          "type": "string"
+        }
+      }
+    },
+    "ReissueCertificateOrderRequest": {
+      "description": "Class representing certificate reissue request.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ReissueCertificateOrderRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate Key Size.",
+              "type": "integer"
+            },
+            "delayExistingRevokeInHours": {
+              "format": "int32",
+              "description": "Delay in hours to revoke existing certificate after the new certificate is issued.",
+              "type": "integer"
+            },
+            "csr": {
+              "description": "Csr to be used for re-key operation.",
+              "type": "string"
+            },
+            "isPrivateKeyExternal": {
+              "description": "Should we change the ASC type (from managed private key to external private key and vice versa).",
+              "type": "boolean"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RenewCertificateOrderRequest": {
+      "description": "Class representing certificate renew request.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "RenewCertificateOrderRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate Key Size.",
+              "type": "integer"
+            },
+            "csr": {
```

Comment:
>csr [](http://example.com/codeflow?start=13&length=3)

Generally prefer using words instead of acronyms/abbreviations (unless it's super-common like IP, PDF, etc). In any case, expand the acronym/abbreviation in the description.